src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/CcdBundleCloningService.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.em.orchestrator.service.caseupdater;
+
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import uk.gov.hmcts.reform.em.orchestrator.service.ccdcallbackhandler.CcdCallbackDto;
+import uk.gov.hmcts.reform.em.orchestrator.service.dto.CcdBundleDTO;
+import uk.gov.hmcts.reform.em.orchestrator.service.dto.CcdValue;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@Transactional
+public class CcdBundleCloningService implements CcdCaseUpdater {
```

Comment:
Why does this implement the CcdCaseUpdater interface? I'm assuming we don't want it to go into the list of handlers for the prepare bundle request

Reply:
Tbh I drew too heavily on stitching service for inspiration. Didn't really look at the handles method. Cleaning this up now